### PR TITLE
Add step for MTA

### PR DESCRIPTION
**Pull request by gabriel-farache**
Add step for MTA

### CODE REVIEW

File: mta/README.md [hunk 1]
```diff
@@ -7,14 +7,19 @@ At the end of a successful assessment workflow, a link to the report will be ava
...
 ## Installation
-Run 
-
+- Run 
 ```console
 helm install mta workflows/mta --namespace=sonataflow-infra
 ```
 
-Verify MTA resources and workflow are ready:
+- Edit the `mtaanalysis-props` confimap to set the `mta.url` with the value of the following command:
```
Comment (masayag) [inline mta/README.md:15]: shouldn't the entry in the configmap be `MTA_URL`?

Comment (gabriel-farache) [reply mta/README.md:15]: well, in the props `confimap` we have the `application.properties` file which contains the property `mta.url` 
This prop can be overridden by an environment variable but the setting of the env var has to take place in the `sonataflow` CR
For the `kustomize` it's different as you only need to change the value in `kustomize/mta/base/config.properties` and then the env var will be set

Comment (gabriel-farache) [reply mta/README.md:15]: I will test but my 1st intuitiion is that it won't work as the props are holding a file, and the patch would replace the whole application.properties if I were to try to set a specific prop in it

But maybe all data in the props CM are mounted as env var?

Comment (masayag) [reply mta/README.md:15]: AFAIK, they should be mounted as files on the pod. have a look at the deployment spec (I'll do the same, but a bit later)

Comment (gabriel-farache) [reply mta/README.md:15]: Yeah so see:
```
$ kubectl patch configmap mtaanalysis-props -p '{"data":{"mta.url":"'$MTA_URL'"}}'
configmap/mtaanalysis-props patched

$oc get cm mtaanalysis-props -o yaml
apiVersion: v1
data:
  application.properties: |
    mta.url = http://mta-ui.openshift-mta.svc.cluster.local:8080
    quarkus.rest-client.mta_json.url = ${mta.url}/hub
    quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.backstage-system/api/notifications/}
    #quarkus.rest-client.mta_json.auth.basicAuth.username = username
    #quarkus.rest-client.mta_json.auth.basicAuth.password = password

    #quarkus.log.category.\"org.apache.http\".level = DEBUG
  mta.url: mta-openshift-mta.apps.cluster-fkhn9.dynamic.redhatworkshops.io
kind: ConfigMap
```
the `mta.url` is added outside of the `application.properties` so it won't be considered by the workflow

**Review by masayag — APPROVED** [review]

File: index.yaml [hunk 2]
```diff
@@ -1,6 +1,30 @@
 apiVersion: v1
 entries:
   workflows:
+  - apiVersion: v2
```
Comment (masayag) [inline index.yaml:4]: pls remove this file from the PR.

File: mta/README.md (hunk 1 above)
Comment (masayag) [reply mta/README.md:15]: My suggestion was to add these commands to the document which is the precise command the user will need to execute:
```
# Retrieve the host from the route named "mta" in the "openshift-mta" namespace
MTA_URL=$(oc -n openshift-mta get route mta -o jsonpath='{.spec.host}')

# Update the value of the key "mta.url" in the ConfigMap "mtaanalysis-props"
kubectl -n <namespace> patch configmap mtaanalysis-props -p '{"data":{"mta.url":"'$MTA_URL'"}}'
```

File: index.yaml (hunk 2 above)
Comment (gabriel-farache) [reply index.yaml]: done

**Review by masayag — APPROVED** [review]

**Pull request merged**